topics/go/language/constants/example3/example3.go

```diff
@@ -14,23 +14,23 @@ func main() {
 		C1 = iota // 2 : Increment by 1
 	)
 
-	fmt.Println("1:", A1, B1, C1)
+	fmt.Printf("1: %d %d %d\n", A1, B1, C1)
```

Comment:
Revert the changes in this file. We don't really even run this code we just have it here to talk about the behavior of constants. IMO students will be distracted by the printf verbs here when I want them just focusing on how constants and iota work.

Reply:
agree